src/common/models/Character.ts

```diff
@@ -0,0 +1,14 @@
+// Would like to keep alphabetized to easily find what we are looking for
+export interface Character {
```

Comment:
General Character interface. I know you started one so if we have conflicts I can help fix them. 

For now, all properties need to be optional (?). character.slice.ts doesn't like it when they are required. Not positive if there is a work around other than creating a blank character. It feels like duplicating work though.